models/chainlink/polygon/chainlink_polygon_ccip_onramps.sql

```diff
@@ -0,0 +1,42 @@
+{{
+    config(
+        tags=['dunesql'],
+        schema = 'chainlink_polygon',
+        alias = alias('ccip_onramps'),
+        materialized = 'incremental',
+        file_format = 'delta',
+        unique_key = ['evt_index', 'evt_tx_hash', 'message'],
+        post_hook='{{ expose_spells(\'["polygon"]\',
+                                "project",
+                                "chainlink",
+                                \'["synthquest"]\') }}'
+    )
+}}
+
+ {% set polygon_router = '0x3c3d92629a02a8d95d5cb9650fe49c3544f69b43' %}
+ {% set test_date = "2022-06-01" %}
+
+select 
+'Polygon Mainnet' as origin
+, 'Ethereum Mainnet' as destination
+, cast(json_extract_scalar(message, '$.sourceChainSelector') as INT256) as source_chain_selector
+, cast(json_extract_scalar(message, '$.sequenceNumber') as INT256)  as sequenceNumber
+, cast(json_extract_scalar(message, '$.feeTokenAmount') as INT256) as feeTokenAmount
```

Comment:
Maybe because of these naming errors:
```suggestion
, cast(json_extract_scalar(message, '$.sequenceNumber') as INT256)  as sequence_number
, cast(json_extract_scalar(message, '$.feeTokenAmount') as INT256) as fee_token_amount
```